outputs.tf

```
@@ -3,10 +3,10 @@
 //
 output "assessment_template_arn" {
   description = "Assessment template ARN"
-  value       = join("", aws_inspector_assessment_template.this.*.arn)
+  value       = aws_inspector_assessment_template.this.*.arn
```

As this is no longer a list, also remove the `.*` list syntax
```suggestion
  value       = aws_inspector_assessment_template.this.arn
```